kong/api/routes/plugins.lua

```diff
@@ -70,14 +70,50 @@ end
 -- cjson can encode the schema.
 local schema_to_jsonable
 do
-  local function fdata_to_jsonable(fdata)
+  local insert = table.insert
+  local ipairs = ipairs
+  local next = next
+
+  local fdata_to_jsonable
+
+  local function fields_to_jsonable(fields)
+    local out = {}
+    for _, field in ipairs(fields) do
+      local fname = next(field)
+      local fdata = field[fname]
+      insert(out, { [fname] = fdata_to_jsonable(fdata, "no") })
+    end
+    setmetatable(out, cjson.empty_array_mt)
```

Comment:
Here, we now that `fields` is always an Array. We should use [cjson.array_mt](https://github.com/openresty/lua-cjson#array_mt) instead, which is is preferred over `empty_array_mt`. The latter is less efficient and, while not documented, is scheduled for deprecation some time in the future since `array_mt` is a better pattern.